CMakeLists.txt

```diff
@@ -13,6 +13,12 @@ else()
 endif()
 
 set(CMAKE_CXX_STANDARD 14)
+#Ensures MSVC treats strings as UTF-8 at both compile/runtime time
+if (CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
```

Comment:
You can use if (MSVC)

https://cmake.org/cmake/help/v3.0/variable/MSVC.html

Reply:
Ok